apps/portal/src/app/connect/pay/customization/connectbutton/page.mdx

```diff
@@ -137,7 +137,7 @@ In some cases, you may only want to show users fiat or crypto payment options fo
 
 ## Theme
 
-You can set the theme for the Pay component, which is set to `"dark"` by default. theme can be set to either `"dark"` , `"light"` or a custom theme object.
+You can set the theme for the Universal Bridge component, which is set to `"dark"` by default. theme can be set to either `"dark"` , `"light"` or a custom theme object.
```

Comment:
This sentence needs to be changed - the theme applies to the entire component and not just the "buy" screen in ConnectButton